computron-slicer_preview_nightly.cmake

```diff
@@ -7,17 +7,17 @@ endmacro()
 
 dashboard_set(DASHBOARDS_DIR        "/D")
 dashboard_set(ORGANIZATION          "Kitware")        # One word, no ponctuation
-dashboard_set(HOSTNAME              "computron-macos-tests")
+dashboard_set(HOSTNAME              "computron")
 dashboard_set(OPERATING_SYSTEM      "macOS")
 dashboard_set(SCRIPT_MODE           "Nightly")        # Experimental, Continuous or Nightly
 dashboard_set(Slicer_RELEASE_TYPE   "P")              # (E)xperimental, (P)review or (S)table
 dashboard_set(WITH_PACKAGES         FALSE)             # Enable to generate packages
 dashboard_set(GIT_TAG               "main")         # Specify a tag for Stable release
 if(APPLE)
-  dashboard_set(CMAKE_OSX_DEPLOYMENT_TARGET "10.13")
+  dashboard_set(CMAKE_OSX_DEPLOYMENT_TARGET "11")
```

Comment:
Is it "11" or "11.0"? Or does it not matter?

Reply:
After the :airplane:  takes off ... I will do some testing and if everything works, I will move forward with integrating this pull-request along with https://github.com/Slicer/Slicer/pull/6695

Reply:
For sake of consistency with the value specified in https://github.com/Slicer/Slicer/pull/6695, using `11.0` makes sense.